test/language/statements/for-in/head-const-bound-names-in-stmt.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: The body may not re-declare variables declared in the head
+negative: SyntaxError
+info: |
```

Comment:
The `>` block literal delimiter is used for "folded" block literals, meaning
newlines will be stripped from the specified value. For the most part, we've
been using newlines to format the contents of our tests' `info` fields. My use
of `|` here is intentional; it may be inconsistent with the existing tests, but
I think correctness trumps consistency.